CHANGES.md

```diff
@@ -7,9 +7,20 @@ Version 0.1.1
 To be released.
 
  -  Relicensed under [LGPL 2.1] or later.
+ -  Added more implicit conversions.  [[#1]]
+     -  `Integer` <-> `short`
+     -  `Integer` <-> `ushort`
+     -  `Integer` -> `int`
+     -  `Integer` <-> `uint`
+     -  `Integer` <-> `long`
+     -  `Integer` <-> `ulong`
+     -  `Binary` -> `bytes`
+     -  `Boolean` <-> `bool`
+     -  `Dictionary<IKey, IValue>` <-> `Bencodex.Types.Dictionary`
+     -  `ImmutableDictionary<IKey, IValue>` <-> `Bencodex.Types.Dictionary`
```

Comment:
How about using proper `↔︎` arrow instead of less-than/greater-than operators?

Reply:
Awesome :+1: